examples/ABL-Atmospheric-Boundary-Layer/input.i3d

```diff
@@ -26,7 +26,7 @@ zlz = 3140.          ! Lz (Size of the box in z-direction)
 ! Boundary conditions
 nclx1 = 0
 nclxn = 0
-ncly1 = 1
+ncly1 = 2
```

Comment:
Is this intentional?

Reply:
I don't know where this is coming from. This is only a legacy example, we need someone that know the BL case se say if make sense or not.

Reply:
The scope of the PR is to split the tests. I think updating the input files is outside of the scope of this PR.

Reply:
I think this is OK since it revert the `input.i3d` to reflect the one of a real case and not the one used for the test. The legacy inputs have been removed since no more relevant.